webview/src/plots/components/styles.module.scss

```diff
@@ -178,6 +197,7 @@ $gap: 20px;
 
 .smallDropIcon {
```

Comment:
Probably just me staring at things too long, but the icon seemed... out of place 🤔. One idea I played with was having the drag and drop icon be centered like targets, but with less opacity, keeping the title the focus: 

https://github.com/iterative/vscode-dvc/assets/43496356/fe2d6908-3cd3-493d-af3a-abce05bf5d27

What do you think?